a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -167,7 +167,7 @@ <h3>Terminology</h3>
 <dt><dfn>electronic braille</dfn></dt>
 <dt><dfn>refreshable braille</dfn></dt>
 <dd>
-<p>The terms "electronic braille" and "refreshable braille" are used interchangeably, which denotes a device with pop-up pins to present the braille on a tactile screen. These devices can be used as a display for a personal computer, or they may be a self-contained multipurpose note taker. </p>
+<p>The terms "electronic braille" and "refreshable braille" can be  used interchangeably, which denotes a device with pop-up pins to present the braille on a tactile screen. These devices can be used as a display attached to  a personal computer, or they may be a self-contained multipurpose note taker with a refreshable braille display. </p>
```

Comment:
This still sounds like a definition of a refreshable braille display. That's different from the generated braille itself, which is what the term is used for in the body. To avoid confusion, we might want to use a term like "dynamic braille" for what gets rendered.